pkgs/top-level/python-packages.nix

```diff
@@ -11914,12 +11914,12 @@ in {
   guessit = callPackage ../development/python-modules/guessit { };
 
   rebulk = buildPythonPackage rec {
```

Comment:
Please also move this expression out of python-packages.nix as described in the header of this file.